Source/Scene/BatchTexture.js

```diff
@@ -32,6 +33,8 @@ export default function BatchTexture(options) {
   Check.typeOf.object("options.owner", options.owner);
   //>>includeEnd('debug');
 
+  this._id = createGuid();
```

Comment:
To answer your question, yes this is correct given that you're storing the batch textures in the statistics object :+1: